src/util/check-supported-platforms.js

```diff
@@ -0,0 +1,7 @@
+export function checkSupportedPlatforms(pkg) {
+  const osList = require(`${pkg}/package.json`).os || [];
```

Comment:
Commented on the zip one as it is actually incorrect to check the OS support for `zip-folder` as it is only used on windows.

The default suggested here probably makes more sense as well @anulman , what do you think?